core/src/main/java/com/linecorp/armeria/common/stream/StreamWriter.java

```diff
@@ -133,4 +135,38 @@ default boolean tryWrite(Supplier<? extends T> o) {
      * signal that the {@link Subscriber} did not consume the stream completely.
      */
     void close(Throwable cause);
+
+    /**
+     * Adapts this {@link StreamWriter} to {@link OutputStream}.
+     *
+     * <p>For example:<pre>{@code
+     * DefaultStreamMessage<String> writer = new DefaultStreamMessage<>();
+     * OutputStream outputStream = writer.toOutputStream(HttpData::toStringUtf8);
+     *
+     * List<String> strings = List.of("foo", "bar", "baz");
+     * for (String string : strings) {
+     *     for (byte b : string.getBytes()) {
+     *         outputStream.write(b);
+     *     }
+     *     outputStream.flush();
+     * }
+     * outputStream.close();
+     *
+     * assert writer.collect().join().equals(List.of("foo", "bar", "baz"));
+     * }</pre>
+     */
+    default OutputStream toOutputStream(Function<? super HttpData, ? extends T> httpDataConverter) {
+        return toOutputStream(httpDataConverter, 8192);
+    }
```

Comment:
We can use the default value in `PathStreamMessage`.
https://github.com/line/armeria/blob/5b384fbe27e7e6f9225d6db91cbb684d09dfbb5e/core/src/main/java/com/linecorp/armeria/common/stream/PathStreamMessage.java#L60
Because if we decided to change the default buffer size, they should be changed together.